### PR TITLE
[MKLDNN] OIW data format support

### PR DESCRIPTION
**Pull request by tsocha**

### CODE REVIEW

File: inference-engine/src/mkldnn_plugin/mkldnn_memory.cpp [hunk 1]
```diff
@@ -579,6 +579,7 @@ MKLDNNMemoryDesc::operator InferenceEngine::TensorDesc() const {
             order = {0, 1};
             blkDims = dims;
             break;
+        case memory::oiw:
```
Comment (ilyachur) [inline inference-engine/src/mkldnn_plugin/mkldnn_memory.cpp:582]: Why do we need support this dormat?
@dmitry-gorokhov Please have a look.

Comment (tsocha) [reply inference-engine/src/mkldnn_plugin/mkldnn_memory.cpp:582]: This change allow to pass an ONNX test case which tests ConvTranspose operator with 1d data.

**Comment by dmitry-gorokhov** [comment]
@tsocha thanks for the patch! 
I have another PR https://github.com/openvinotoolkit/openvino/pull/3725 which totally reworked this part of the plug-in. Seems like it allows ConvTranspose test to pass. I am going to merge https://github.com/openvinotoolkit/openvino/pull/3725 in couple of days so not sure we need to merge this one.

**Comment by tsocha** [comment]
Yeah you are correct.
I'm closing this PR since convtranspose_1d onnx test has been fixed in the more complex PR #3725

**Pull request closed without merging**